app/src/main/java/org/kiwix/kiwixmobile/database/BookDao.java

```diff
@@ -17,17 +17,17 @@
 
 public class BookDao {
   private KiwixDatabase mDb;
-
+  private int FLAG;
```

Comment:
I am unable to understand the use of this FLAG. Can you please explain in a few lines ?

Reply:
@srv-twry In each those pairs differs by some lines of code and since all the code was redundant, i though to move the complete code to another function and check for flag for those different lines of code.

Reply:
@srv-twry should i go about it without flag?

Reply:
@srv-twry  please refer to the latest commit below.